pytorch_lightning/core/hooks.py

```diff
@@ -426,6 +429,22 @@ def train_dataloader(self):
                 )
                 return loader
 
+            # multiple dataloaders, return as list
+            def train_dataloader(self):
+                mnist = MNIST(...)
+                cifar = CIFAR(...)
+                # each batch will be a list of tensors: [batch_mnist, batch_cifar]
+                return [torch.utils.data.DataLoader(dataset=mnist, batch_size=self.batch_size, shuffle=True),
+                        torch.utils.data.DataLoader(dataset=cifar, batch_size=self.batch_size, shuffle=True)]
```

Comment:
Could we shorten or format so that the line length is a bit shorter? The generated docs wrap around which is less readable

![image](https://user-images.githubusercontent.com/5495193/108597318-b8e9ab00-7388-11eb-972a-cff75f068197.png)

Reply:
Done :]